libs/dev-dock/backend/src/dev_dock_api.ts

```diff
@@ -168,6 +169,14 @@ function buildApi(devDockFilePath: string, machineType: MachineType) {
       usbHandler.clearData();
     },
 
+    async captureScreenshot({ appName }: { appName: string }): Promise<string> {
+      assert(/^[a-z0-9]+$/i.test(appName));
+      const downloadsPath = join(homedir(), 'Downloads');
+      const fileName = `Screenshot-${appName}-${new Date().toISOString()}.png`;
+      await execFile('gnome-screenshot', ['-f', join(downloadsPath, fileName)]);
+      return fileName;
+    },
```

Comment:
One of the nice features about the previous screenshot functionality was that it captured the browser window only when using device emulation, which is important for screenshotting UI since you want to have the proper screen dimensions. I'm assuming `gnome-screenshot` takes a full-screen picture?

Reply:
Added this feature back using the same approach, so we still require `kiosk.captureScreenshot`. The only difference is that it's the backend that writes the file rather than the frontend. This lets us remove `kiosk.saveAs`.